docs/Catalog/Embed.md

```diff
@@ -0,0 +1,206 @@
+The Quilt catalog has a special (limited) S3 browser that can be embedded into any web app using an iframe.
+This embeddable browser is served off the main catalog server under `/__embed` route.
+
+![](../imgs/embed-example.png)
+
+
+## Usage
+
+Here's an example code written in plain JavaScript + React outlining the basic use cases:
+
+```js
+import * as React from 'react'
+
+const EMBED_ORIGIN = 'https://my-quilt-catalog'
+const PARENT_ORIGIN = window.location.origin
+const EVENT_SOURCE = 'quilt-embed'
+
+const mkNonce = () => `${Math.random()}`.slice(2)
+
+function Embed() {
+  const iframeRef = React.useRef(null)
+
+  const [nonce, setNonce] = React.useState(mkNonce)
+
+  // nonce is used to identify "our" embed instance and make sure we're receiving messages from the same instance
+  // origin must be sent as a query parameter to enable cross-origin message passing from embed to the parent
+  const src = `${EMBED_ORIGIN}/__embed?nonce=${nonce}&origin=${encodeURIComponent(PARENT_ORIGIN)}`
+
+  const postMessage = React.useCallback(
+    // function for sending messages to the embed (it will only handle messages sent by the window that opened it aka parent)
+    (msg) => {
+      if (!iframeRef.current) return
+      // origin must be sent as a second parameter to enable cross-origin message passing
+      iframeRef.current.contentWindow.postMessage(msg, EMBED_ORIGIN)
+    },
+    [iframeRef],
+  )
+
+  const initialize = React.useCallback(() => {
+    postMessage({
+      type: 'init',
```

Comment:
Maybe highlight in comments which parameters are required and which are optional

Reply:
i've added this info to the API reference section